examples/client/getPassiveDelegationInfo.ts

```diff
@@ -0,0 +1,52 @@
+import { PassiveDelegationStatus } from '@concordium/common-sdk';
+import { createConcordiumClient } from '@concordium/node-sdk';
+import { credentials } from '@grpc/grpc-js';
+
+import meow from 'meow';
+
+const cli = meow(
+    `
+  Usage
+    $ yarn ts-node <path-to-this-file> [options]
+
+  Options
+    --help,     -h  Displays this message
+    --block,    -b  A block to query from, defaults to last final block
+    --endpoint, -e  Specify endpoint of the form "address:port", defaults to localhost:20000
+`,
+    {
+        importMeta: import.meta,
+        flags: {
+            endpoint: {
+                type: 'string',
+                alias: 'e',
+                default: 'localhost:20000',
+            },
+            block: {
+                type: 'string',
+                alias: 'b',
+                default: '', // This defaults to LastFinal
+            },
+        },
+    }
+);
+
+const [address, port] = cli.flags.endpoint.split(':');
+const client = createConcordiumClient(
+    address,
+    Number(port),
+    credentials.createInsecure(),
+    { timeout: 15000 }
+);
+
+if (cli.flags.h) {
+    cli.showHelp();
+}
+
+/// Retrieves information about the passive delegators, at the end of the specified block.
+(async () => {
+    const bakerPoolInfo: PassiveDelegationStatus =
+        await client.getPassiveDelegationInfo(cli.flags.block);
+
+    console.dir(bakerPoolInfo, { depth: null, colors: true });
```

Comment:
```suggestion
    const passiveDelegationInfo: PassiveDelegationStatus =
        await client.getPassiveDelegationInfo(cli.flags.block);

    console.dir(passiveDelegationInfo, { depth: null, colors: true });
```

Also missing something like this: :upside_down_face: 
`const peerVersion: string = nodeInfo.peerVersion;`